datafusion/src/sql/planner.rs

```diff
@@ -2641,13 +2701,23 @@ mod tests {
     }
 
     #[test]
-    fn over_not_supported() {
+    fn empty_over() {
         let sql = "SELECT order_id, MAX(order_id) OVER () from orders";
-        let err = logical_plan(sql).expect_err("query should have failed");
-        assert_eq!(
-            "NotImplemented(\"Unsupported OVER clause ()\")",
-            format!("{:?}", err)
-        );
+        let expected = "\
+        Projection: #order_id, #MAX(order_id)\
+        \n  WindowAggr: windowExpr=[[MAX(#order_id)]] partitionBy=[], orderBy=[]\
+        \n    TableScan: orders projection=None";
+        quick_test(sql, expected);
+    }
+
+    #[test]
+    fn empty_over_plus() {
```

Comment:
I would like to see some additional tests to increase the coverage:
1. Multiple window functions in the same query (as the frontend code supports this)
2. Built in window functions (e.g `ROW_NUMBER()` and `row_number()` and `lead(col)`)
3. A test in `tests/sql.rs` that runs (and throw as "NotYetImplemented") error for queries that have an `OVER()` clause (to exercise the physical planning logic)

Reply:
done